dimagi/pages/templates/sections/commcare/pricing/plans/_base.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 
 <div class="column" id="{{ css_id }}">
-  <a class="tile {{ theme }}-border plans-tile{% if highlight %} plan-highlight{% endif %} {{ theme }}-theme {{ click_class }}"
```

Comment:
did we also take care of where the `highlight` variable is set for the template :)

Reply:
Updated.
Should we need to remove the CSS for this?

Reply:
@hemanthReddy06 hmm, that's a good question. I think we're fine at least for now to just get it out